libs/framework/include/celix_constants.h

```diff
@@ -158,19 +158,54 @@ extern "C" {
 #define OSGI_FRAMEWORK_IMPORT_LIBRARY CELIX_FRAMEWORK_IMPORT_LIBRARY
 
 /**
- * @brief Celix framework environment property (named "org.osgi.framework.storage") specifying the cache
+ * @brief Celix framework environment property (named "CELIX_FRAMEWORK_CACHE_DIR") specifying the cache
  * directory used for the bundle caches.
  *
- * If not specified ".cache" is used.
+ * This property is not used if CELIX_FRAMEWORK_CACHE_USE_TMP_DIR=true.
+ *
+ * Default value is ".cache".
+ */
+#define CELIX_FRAMEWORK_FRAMEWORK_CACHE_DIR "CELIX_FRAMEWORK_CACHE_DIR"
+
+//@deprecated use CELIX_FRAMEWORK_FRAMEWORK_CACHE_DIR
+#define CELIX_FRAMEWORK_FRAMEWORK_STORAGE   "CELIX_FRAMEWORK_CACHE_DIR"
+
+//@deprecated use CELIX_FRAMEWORK_FRAMEWORK_CACHE_DIR
+#define OSGI_FRAMEWORK_FRAMEWORK_STORAGE    "org.osgi.framework.storage"
```

Comment:
I notice there are still `OSGI_FRAMEWORK_FRAMEWORK_STORAGE` user pubsub/rsa tests

Reply:
Documentation of `celix_bundleCache_create` has not been updated.